clang/include/clang/Basic/Attr.td

```diff
@@ -1415,13 +1415,33 @@ def LoopUnrollHint : StmtAttr {
 }
 
 def IntelReqdSubGroupSize: InheritableAttr {
-  let Spellings = [GNU<"intel_reqd_sub_group_size">,
-                   CXX11<"intel", "reqd_sub_group_size">];
+  let Spellings = [
+    GNU<"intel_reqd_sub_group_size">, CXX11<"intel", "reqd_sub_group_size">,
+    CXX11<"intel", "sub_group_size"> // SYCL2020 spelling.
+  ];
```

Comment:
will we deprecated old spelling here?

Reply:
I think we should, though I considered it a separate decision/effort from this patch.

Reply:
Yes, agree.